plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -33,7 +33,7 @@ return {--do NOT change this line
 
  -- LIST YOUR FEEDS HERE:
 
- { "http://feeds.reuters.com/Reuters/worldNews?format=xml", limit = 2, download_full_article=true, include_images=true, enable_filter=true},
+ { "https://ourworldindata.org/atom.xml", limit = 2, download_full_article=true, include_images=true, enable_filter=true},
```

Comment:
Is this change accidental for testing or purposeful?

Reply:
Oh, that's accidental for testing. 

The white spaces are also getting in the way of my sanity. I'm so sorry. The Lua setup I had on Emacs was/is kind of janked out.